install/kubernetes/cilium/templates/spire/agent/clusterrole.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if and .Values.auth.mTLS.spire.enabled .Values.auth.mTLS.spire.install.enabled -}}
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: spire-agent
```

Comment:
given this is a cluster role thus not namespaced we should be good citizens and either prefix it cilium- or make it configurable (or both)